recipes/coin-clp/all/conanfile.py

```diff
@@ -1,14 +1,15 @@
 from conan import ConanFile
 from conan.errors import ConanInvalidConfiguration
-from conan.tools.files import get, apply_conandata_patches, mkdir, rm, rmdir, rename
```

Comment:
Please try to no move things so much, it makes the diff bigger and less likely to be reviewed with the huge backlog 🙏